### PR TITLE
Rename PowershellPreparer to EnvironmentVariablesLoader

### PR DESCRIPTION
**Pull request by lmazuel**

### CODE REVIEW

File: tools/azure-sdk-tools/devtools_testutils/__init__.py [hunk 1]
```diff
@@ -60,6 +61,7 @@
     "RandomNameResourceGroupPreparer",
     "CachedResourceGroupPreparer",
     "PowerShellPreparer",
```
Comment (xiangyan99) [inline tools/azure-sdk-tools/devtools_testutils/__init__.py:63]: Remove?

**Comment by kristapratico** [comment]
For those of us that already use this in our test code, should we open separate PRs to address the breaking name change?

**Comment by kristapratico** [comment]
Our dev docs reference PowershellPreparer. Maybe we should update these too. 

https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/tests.md#create-live-test-resources
https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/test_proxy_migration_guide.md

File: tools/azure-sdk-tools/devtools_testutils/__init__.py (hunk 1 above)
Comment (rakshith91) [reply tools/azure-sdk-tools/devtools_testutils/__init__.py:63]: i'd say keep this till next MQ for backward compat (i'm assuming hence the alias)

**Review by rakshith91 — APPROVED** [review]

**Comment by mccoyp** [comment]
I can update the developer docs that mention the preparer once we've decided on a name, because it looks like the docs could use an overall revamping.

This is nitpicky, but what do people think about "EnvironmentVariableLoader" instead of "EnvironmentVariable**s**Loader"? (I do think "Loader" is more appropriate than "Preparer" since it only fetches vars, and possibly sanitizes them)

**Review by mccoyp — APPROVED** [review]

**Pull request merged**